### PR TITLE
Adding additional info on upgrade note

### PR DESCRIPTION
**Pull request by bgaydosrh**
For https://bugzilla.redhat.com/show_bug.cgi?id=1796446
This PR replaces https://github.com/openshift/openshift-docs/pull/19441

@dankenigsberg please code review
@alexxa please QE review

Please label Peer Review Needed and cherry-pic to enterprise-4.3.

Thanks
Bob

### CODE REVIEW

File: cnv/cnv-release-notes/cnv-release-notes.adoc [hunk 1]
```diff
@@ -42,8 +42,10 @@ This issue has been fixed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=171
 
 == Known issues
 
-* If you have {CNVProductName} 2.1.0 deployed, you must first upgrade to 2.2.0
-before upgrading {product-title}.
+* If you have {CNVProductName} 2.1.0 deployed, you must first upgrade {CNVProductName}
+to 2.2.0 before upgrading {product-title}. Upgrading {product-title} before upgrading
+{CNVProductName} may trigger virtual machine deletion.
```
Comment (ousleyp) [inline cnv/cnv-release-notes/cnv-release-notes.adoc]: s/may/might

Comment (bgaydosrh) [reply cnv/cnv-release-notes/cnv-release-notes.adoc]: Fixed.

**Comment by dankenigsberg** [comment]
fine by me!

**Comment by bgaydosrh** [comment]
> fine by me!

Awesome, should be in shortly.

**Comment by bgaydosrh** [comment]
Also added fix for https://bugzilla.redhat.com/show_bug.cgi?id=1797572 and changed # to $ for the oc command per our standard notation.

**Review by alexxa — APPROVED** [review]

**Comment by bgaydosrh** [comment]
Final test build for peer review: http://file.bos.redhat.com/bgaydos/020320_2/cnv/cnv-release-notes/cnv-release-notes.html

File: cnv/cnv-release-notes/cnv-release-notes.adoc [hunk 2]
```diff
@@ -42,8 +42,10 @@ This issue has been fixed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=171
 
 == Known issues
 
-* If you have {CNVProductName} 2.1.0 deployed, you must first upgrade to 2.2.0
-before upgrading {product-title}.
+* If you have {CNVProductName} 2.1.0 deployed, you must first upgrade {CNVProductName}
+to 2.2.0 before upgrading {product-title}. Upgrading {product-title} before upgrading
+{CNVProductName} might trigger virtual machine deletion.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1785661[*BZ#1785661*]).
```
Comment (ousleyp) [inline cnv/cnv-release-notes/cnv-release-notes.adoc]: super tiny nit: take out the period after the BZ link to match the others.

Comment (bgaydosrh) [reply cnv/cnv-release-notes/cnv-release-notes.adoc]: Removed.

**Pull request merged**